README.md

```diff
@@ -43,6 +43,16 @@ module "default-backend-web-app" {
   alb_ingress_healthcheck_path = "/healthz"
   alb_ingress_paths            = ["/*"]
   codepipeline_enabled         = "false"
+  environment = [
```

Comment:
just a nitpick, but this TF snippet needs `terraform fmt`
well be regenerated after updating in `README.yaml` and then rebuilding README

```
make init
make readme/deps
make readme
```